src/notebook.h

```diff
@@ -434,6 +434,13 @@ class MKCAL_EXPORT Notebook
     */
     int flags() const;
 
+    /**
+      Get the default flag value.
+
+      This utility function mostly facilitates easy database modification.
+    */
+    static int defaultFlag();
```

Comment:
Hmh, "The content of flags is not public as such." and then this exposes one flag. Could be nice if this was possible to hide a bit from the public api, though that applies to the existing flags() thing too.

Reply:
I was thinking to make these routines private but adding the ExtendedCalendar as a friend. What do you think ?